src/utilities/theme/role-variants.ts

```diff
@@ -4,13 +4,13 @@ import {RoleVariants} from './types';
 export enum UNSTABLE_Color {
   Surface = '#FAFAFA',
   DarkSurface = '#111213',
-  OnSurface = '#1F2225',
-  Interactive = '#0870D9',
-  Neutral = '#EAEAEB',
+  OnSurface = '#111213',
```

Comment:
This change to `OnSurface` means we're getting failures for the `textOnSurfaceDark` variant. That variant is used in all `destructive` and `primary` buttons as a text value. Those are currently failing the Pally accessibility check for color contrast. We should be able to solve that by making the lightness a bit higher for that variant.

Reply:
I'll have to set up a call... because I'm having a hard time connecting the dots on that one now.

Reply:
Sorry this was a bit confusing. I ultimately arrived at the conclusion that we should be using a different variant for setting the text color on primary and critical actions.